frontend/src/scenes/heatmaps/HeatmapsBrowser.tsx

```diff
@@ -70,7 +98,18 @@ export function HeatmapsBrowser(): JSX.Element {
                                     <AuthorizedUrlList type={AuthorizedUrlListType.TOOLBAR_URLS} />
                                 </div>
                             ) : (
-                                <>
+                                <div className="flex flex-row space-x-2 w-full">
```

Comment:
fyi @pauldambra - always use gap-N if you are flexing. space-N is more of a helper hack using margins but can cause weird things (like the spinner overlay having a margin applied)